pipelines/reconfigure.yml

```diff
@@ -296,16 +296,19 @@ jobs:
     vars:
       branch_name: algorithm-v3
       environment: algorithm
+      compatibility_version: 6.0.0
   - set_pipeline: runtime-drills
     file: pipelines-and-tasks/pipelines/branch.yml
     vars:
       branch_name: master
       environment: runtime-drills
+      compatibility_version: dev
   - set_pipeline: set-zstd-drills-env
     file: pipelines-and-tasks/pipelines/branch.yml
     vars:
       branch_name: use_zstd
       environment: use-zstd
+      compatibility_version: 6.0.0
```

Comment:
I don't know if these numbers are accurate, they seemed to all be paused on CI and were never run at all. 

I'm assuming `algorithms-v3` is v6, `runtime-drills` seems to have replaced `drills.yml` (not sure why the file is still in this repo) and I have no idea what `use-zstd` is